models/public/pspnet-pytorch/pspnet-pytorch.md

```diff
@@ -0,0 +1,73 @@
+# pspnet-pytorch
+
+## Use Case and High-Level Description
+
+pspnet-pytorch is a semantic segmentation model, pretrained on [Pascal VOC](http://host.robots.ox.ac.uk/pascal/VOC/) dataset for 21 object classes. The model was built on [ResNetV1-50](https://arxiv.org/pdf/1812.01187.pdf) backbone and PSP segmentation head. This model is used for pixel-level prediction tasks. For details see [repository](https://github.com/open-mmlab/mmsegmentation/tree/master).
```

Comment:
I think article for PSP Network is also should be added https://arxiv.org/abs/1612.01105

Reply:
I also not sure that you use correct link on ResNetV1-50 backbone, because originally PSPNet was designed in 2017, while aboth article release only in 2018

Reply:
mmsegmentation uses Resnet-50c and refers to this paper https://github.com/open-mmlab/mmsegmentation/blob/993be2523b908d1e0cb45bc68e92d210b5b2cda1/mmseg/models/backbones/resnet.py#L669